spec/features/feedex_spec.rb

```diff
@@ -107,122 +105,117 @@
-          },
-          {
-            type: "problem-report",
-            path: "/guidance/doing-bad-things",
-            url: "http://www.dev.gov.uk/guidance/doing-bad-things",
-            created_at: DateTime.parse("2013-02-01"),
-            what_doing: "looking at what you consider bad",
-            what_wrong: "so many typos, whoever wrote this must have been angry",
-            referrer: "https://www.gov.uk/having-fun",
-          },
-        ]
-      }
+      "current_page" => 1,
+      "pages" => 1,
+      "page_size" => 2,
+      "results" => [
+        {
+          type: "problem-report",
+          path: "/guidance/doing-fun-things",
+          url: "http://www.dev.gov.uk/guidance/doing-fun-things",
+          created_at: DateTime.parse("2013-02-01"),
+          what_doing: "finding out about fun",
+          what_wrong: "no mention of petting a dog",
+          referrer: "https://www.gov.uk/having-fun",
+        },
+        {
+          type: "problem-report",
+          path: "/guidance/doing-bad-things",
+          url: "http://www.dev.gov.uk/guidance/doing-bad-things",
+          created_at: DateTime.parse("2013-02-01"),
+          what_doing: "looking at what you consider bad",
+          what_wrong: "so many typos, whoever wrote this must have been angry",
+          referrer: "https://www.gov.uk/having-fun",
+        },
+      ]
     )
 
     path_feedback_request = stub_support_api_anonymous_feedback(
       { path_prefix: '/guidance' },
-      {
-        "current_page" => 1,
-        "pages" => 1,
-        "page_size" => 2,
-        "results" => [
-          {
-            type: "problem-report",
-            path: "/guidance/doing-fun-things",
-            url: "http://www.dev.gov.uk/guidance/doing-fun-things",
-            created_at: DateTime.parse("2013-02-01"),
-            what_doing: "finding out about fun",
-            what_wrong: "no mention of petting a dog",
-            referrer: "https://www.gov.uk/having-fun",
-          },
-          {
-            type: "problem-report",
-            path: "/guidance/doing-bad-things",
-            url: "http://www.dev.gov.uk/guidance/doing-bad-things",
-            created_at: DateTime.parse("2013-02-01"),
-            what_doing: "looking at what you consider bad",
-            what_wrong: "so many typos, whoever wrote this must have been angry",
-            referrer: "https://www.gov.uk/having-fun",
-          },
-          {
-            type: "problem-report",
-            path: "/guidance/wearing-a-hat",
-            url: "http://www.dev.gov.uk/guidance/wearing-a-hat",
-            created_at: DateTime.parse("2013-02-01"),
-            what_doing: "want to know about balaclavas",
-            what_wrong: "I know they're not really a hat, but would it hurt to mention them",
-            referrer: "https://www.gov.uk/hats",
-          },
-        ]
-      }
+      "current_page" => 1,
+      "pages" => 1,
+      "page_size" => 2,
+      "results" => [
+        {
```

Comment:
Same as before, I think this might be shortened by assigning this has once and then re-using the it in multiple scenarios, since the only that really needs to vary afaik is the value of the params received in the first hash (e.g. `{ path_prefix: '/guidance' }` )

Reply:
You're right that the only thing that needs to vary is the args in the query hash, and the actual results don't matter. However, I am a fan of making sure that test data is realistic (because we never know how the app might change in the future and our tests break if the data isn't correct).  You're right that it's quite noisy though, I'll pull out these stubs to methods and see if I can make the test shorter.

Reply:
Did this in fb09e4378cd7233332333cf50f253abbcbe5ff89